index.html

```diff
@@ -76,6 +76,271 @@ <h1>I’m Joseph Glad to see you!</h1>
+            <li>
+              <a href="#"
+                ><img
+                  src="/img/icons/angellist.svg"
+                  alt="angellist"
+                  title="follow me on"
+              /></a>
+            </li>
+          </ul>
+          <a href="#" class="btn btn-enabled">Get my resume</a>
+        </div>
+
+        <div class="about-me__tools">
+          <div class="tools prog-languages">
+            <h3>languages</h3>
+            <img
+              src="/img/down-chevron.png"
+              alt="drop down to see the languages"
+            />
+          </div>
+          <ul class="programming-lang-container">
+            <li class="programming-lang">
+              <img src="/img/js.png" alt="javascript" />
+              <a href="#">javascript</a>
+            </li>
+            <li class="programming-lang">
+              <img src="/img/css.png" alt="CSS" />
+              <a href="#">CSS</a>
+            </li>
+            <li class="programming-lang">
+              <img src="/img/html.png" alt="HTML" />
+              <a href="#">HTML</a>
+            </li>
+          </ul>
+          <div class="tools frameworks">
+            <h3>Frameworks</h3>
+            <img
+              src="/img/forward-chevron.png"
+              alt="drop down to see the frameworks"
+            />
+          </div>
+          <div class="tools skils">
+            <h3>Skills</h3>
+            <img
+              src="/img/forward-chevron.png"
+              alt="drop down to see the skils"
+            />
+          </div>
+        </div>
+      </section>
+      <section class="contact-form">
+        <div class="contact-form__intro">
+          <h2>Contact me</h2>
+          <p>
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <div class="contact-form__details">
+          <form
+            action="https://formspree.io/f/mrgjbnol"
+            method="post"
+            class="form-group"
+          >
+            <input
+              type="text"
+              name="name"
+              class="form-control"
+              id="name"
+              size="30"
+              placeholder="Joseph"
+              required
+            />
+            <input
+              type="email"
+              name="email"
+              class="form-control"
+              id="email"
+              placeholder="drack.sir01@gmail.com"
+              required
+            />
+            <textarea
+              name="message"
+              class="form-control text-area"
+              id="message"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              placeholder="Write your message here"
+              required
+            ></textarea>
+            <button class="btn btn-enabled" type="button">Get It Touch</button>
```

Comment:
- [ ] The button doesn't work. When I click on the button the form should be submitted. Please, try to change the type of the button to `submit`